src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutes.scala

```diff
@@ -222,13 +222,13 @@ class ResourceRoutes(resources: Resources[Task])(implicit cache: DistributedCach
 
   private def listings(implicit token: Option[AuthToken]): Route =
     (pathPrefix("resources") & project) { implicit wrapped =>
-      (get & parameter('deprecated.as[Boolean].?) & paginated & hasPermission(resourceRead)) {
+      (get & parameter('deprecated.as[Boolean].?) & paginated & hasPermission(resourceRead) & pathEndOrSingleSlash) {
         (deprecated, pagination) =>
           val results = cache.views(wrapped.ref).flatMap(v => resources.list(v, deprecated, pagination))
           trace("listResources") {
             complete(results.runAsync)
           }
-      } ~ pathPrefix(aliasOrCurie)(listings)
+      } ~ (pathPrefix(aliasOrCurie) & pathEndOrSingleSlash)(listings)
```

Comment:
pathEndOrSingleSlash shouldn't go here. It has to be handled by the `listings` method